deploy/README.md

```diff
@@ -0,0 +1,21 @@
+# Convox Deploy Action
+This Action [Deploys](https://docs.convox.com/introduction/getting-started#deploy-your-application) an app based on a [convox.yml](https://docs.convox.com/application/convox-yml) to Convox. The Deploy action performs the functions of combining the [Build](../build) and [Promote](../promote) actions but in a single step.
+
+## Inputs
+### `password`
```

Comment:
Should probably pull password and host out of the docs here and let's see if people actually need it. Deploy is already a bit confusing since it does the same thing as other actions.